functional_tests/test_find_bugs_blocker.py

```diff
@@ -8,14 +8,11 @@ def test_find_bugs_blocker(self):
         out = subprocess.check_output(
             constants.ELLIOTT_CMD
             + [
-                "--assembly=stream", "--group=openshift-4.3", "find-bugs:blocker", '--exclude-status=ON_QA'
+                "--assembly=stream", "--group=openshift-4.6", "find-bugs:blocker", '--exclude-status=ON_QA'
```

Comment:
Could we take the most recent future EUS release (4.10 i think)? Then we can forget about this the longest time :)

Reply:
agreed but i do not want to hold this PR waiting on that...